aries_vcx/tests/utils/devsetup_faber.rs

```diff
@@ -58,48 +49,23 @@ pub async fn create_faber_trustee(genesis_file_path: String) -> Faber {
 }
 
 pub async fn create_faber(genesis_file_path: String) -> Faber {
-    let (public_did, wallet_handle) = dev_setup_wallet_indy(&generate_random_seed()).await;
-    let wallet = Arc::new(IndySdkWallet::new(wallet_handle));
-    let profile = dev_build_featured_profile(genesis_file_path.clone(), wallet).await;
-    profile
-        .inject_anoncreds()
-        .prover_create_link_secret(DEFAULT_LINK_SECRET_ALIAS)
-        .await
-        .unwrap();
-    Faber::setup(profile, genesis_file_path, public_did).await
+    create_faber_from_seed(&generate_random_seed(), genesis_file_path).await
 }
 
 impl Faber {
```

Comment:
The `create_schema` is only used in 2 tests,  `pairwise_info` seems to be unused (?) - seems like fairly easy win to remove these 2 as well.
Then the only difference between Alice and Faber would be `institution_did`, which is used quite heavily. 

But it could be another PR, just noting

Reply:
This is exactly what I am doing right now :)